@xen-orchestra/web-core/lib/components/ui/input/UiTextarea.vue

```diff
@@ -0,0 +1,111 @@
+<!-- v2 -->
+<template>
+  <div class="ui-textarea">
+    <UiLabel v-if="slots.default" :accent="labelAccent" :required :icon :href><slot /></UiLabel>
+    <textarea v-model="model" :class="accent" :disabled class="textarea" v-bind="$attrs" />
```

Comment:
Regarding the `accent`, two things:
- I think it would be better to add the `accent` class to the root element (even if the document needs to be updated, [this rule](https://github.com/vatesfr/xen-orchestra/blob/master/%40xen-orchestra/web-core/docs/guidelines/component-variants.md#variant-classes) still does make sense : `The root element will then have secondary classes which reflect the current variants being applied`
- Since it is a variant of the component, I think we could also apply the other rule : `The pattern for these classes is <prop-name>--<prop-value>`. So in this case it would be `accent--info` for example, and this class could be created by using `toVariants({ accent })`

So, in the template, we could have the first `div` as:
```html
<div class="ui-textarea" :class="toVariants({ accent })">
```

And the CSS needs to be updated to something like this:
```css
.ui-textarea {
  display: flex;
  flex-direction: column;
  gap: 0.4rem;

  &.accent--info {
    .textarea {
      border-color: var(--color-neutral-border);

      &:hover {
        border-color: var(--color-info-item-hover);
      }

      &:active {
        border-color: var(--color-info-item-active);
      }

      &:focus {
        border-width: 0.2rem;
        border-color: var(--color-info-item-base);
      }

      &:disabled {
        background-color: var(--color-neutral-background-disabled);
        border-color: var(--color-neutral-border);
      }
    }
  }
```
And so on for the other variants. What do you think about that?
I know it would add a few more lines in the CSS, but I think it reflects more the logic "the root element is in charge of handling the variants", and "the children elements are altered by the variants". And it would keep consistency with the other components of the design system.